examples/sparse/gcn.py

```diff
@@ -0,0 +1,92 @@
+"""
+[Semi-Supervised Classification with Graph Convolutional Networks](https://arxiv.org/abs/1609.02907)
+"""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.data import CoraGraphDataset
+from dgl.mock_sparse import create_from_coo, diag, identity
+from torch.optim import Adam
+
+class GCN(nn.Module):
+    def __init__(self, A, in_size, out_size, hidden_size=16):
```

Comment:
I prefer passing `A` in `forward` to be consistent with other examples.

Reply:
Can you also address this comment?

Reply:
I change the train function into train(g, model, A, X), but don't actually use it